docs/docs/recipes/protect-your-api/README.md

```diff
@@ -2,4 +2,269 @@
 sidebar_position: 2
 ---
 
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
 # ⚔️ Protect your API
+
+Along with a fluent user sign-in experience, by default, Logto comes with the [API resources](../../references/resources/README.md) authorization service. It will help you protect the private APIs resources from anonymous identities. Let's walk through the following steps and implement the API resource authorization guard of your own using Logto.
+
+:::info
+For more details, please check out our [API Resources doc](../../references/resources/README.md)
+:::
+
+## 1. Register the API resource through Logto **Admin Console**
```

Comment:
maybe we don't need the prefix `1.` since they are listed in order on the right-side TOC

Reply:
updated